src/Core/src/Platform/Windows/SearchBarExtensions.cs

```diff
@@ -112,6 +113,16 @@ public static void UpdateMaxLength(this AutoSuggestBox platformControl, ISearchB
 			if (maxLength == -1)
 				maxLength = int.MaxValue;
 
+			var children = platformControl.GetChildren<TextBox>();
+			if (children is not null)
+			{
+				var textBox = children.FirstOrDefault();
```

Comment:
By convention we stay away from using Linq

Can you remove the use of linq here?